fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/postgres/PostgresDoesTableExist.java

```diff
@@ -46,8 +46,8 @@ public Boolean run(IDatabaseTranslator translator, Connection c) {
         final String sql =
                 "  SELECT 1 "
                 + "  FROM information_schema.tables "
-                + " WHERE LOWER(table_schema) = LOWER('" + schemaName + "') "
-                + "   AND LOWER(table_name) = LOWER('" + tableName + "')";
+                + " WHERE table_schema = LOWER('" + schemaName + "') "
+                + "   AND table_name = LOWER('" + tableName + "')";
```

Comment:
I think initially our Postgres code allowed upper case input. I know Postgres smacks it to LOWER internally.
Is this truly backwards compatible with our older versions?

Please confirm

Reply:
PostgreSQL lower-cases identifiers (like table-names) unless they are quoted (which we don't...because it makes things ugly and unreadable). Also, putting LOWER on the columns in that way prevents use of any indexes which could lead to much worse performance for bigger schemas.